packages/frameworks/index.d.ts

```diff
@@ -0,0 +1,16 @@
+export interface FrameworkDetectionItem {
+  file: string;
+  matchContent?: string;
+}
+
+export interface Framework {
+  name: string;
+  slug: string;
+  logo: string;
+  tagline: string;
+  website: string;
+  detectors?: {
+    every?: FrameworkDetectionItem[];
+    some?: FrameworkDetectionItem[];
+  };
+}
```

Comment:
Is this being type checked at build time or are you manually matching this to the json file?

If its manual, can we rename the `frameworks.json` to `fameworks.ts` and generate this `.d.ts` file?

Reply:
I agree that the `framework.json` file should come from a .ts file, but I think we need a `.json` file in the repo, at least it looks like it's using the repo url for the `logo` data.

In any case I'd like to update them in a different PR, they are only used for tests at the moment

Reply:
> at least it looks like it's using the repo url for the logo data.

Can you explain more? What is using the repo url?

Reply:
I see now, its used in an API that front consumes.